common/views/components/Iframe/Iframe.tsx

```diff
@@ -137,7 +137,7 @@ class Iframe extends Component<Props, State> {
                 medium: 1,
                 small: 1,
               }}
-              quality={60}
+              quality={45}
```

Comment:
Is it worth pulling this out into a named constant so we can be consistent across the site?

Reply:
yep, that's coming in another PR